src/main/java/com/pollapp/entity/Category.java

```diff
@@ -0,0 +1,30 @@
+package com.pollapp.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "category")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private String name;
+
```

Comment:
Aren't we missing a constructor?

Reply:
I did not add a constructor because I did not need it. Default constructor will be generated automatically. It can be also added later if needed